src/google-maps/google-map/google-map.ts

```diff
@@ -249,6 +258,9 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
 
   ngOnChanges() {
     this._setSize();
+    if (this._googleMap && this.mapTypeId) {
+      this._googleMap.setMapTypeId(this.mapTypeId);
```

Comment:
This will call setMapTypeId for every change. My preference would be to continue the existing pattern, or failing that, at least include a check that the mapTypeId has changed.

Reply:
That's correct, but from what I can tell, the Google Maps API already knows not to do anything if you're giving it the same map type that it already has.

Reply:
Ok, sounds good